src/haxelib/server/Update.hx

```diff
@@ -0,0 +1,61 @@
+package haxelib.server;
+
+import haxelib.server.SiteDb;
+
```

Comment:
I think we need to use transactions for the public methods of the `Update` class. There are mulpile replicas of haxe lib server pod running in the cluster.

Reply:
Also, both dev (development-lib.haxe.org) and prod (lib.haxe.org) connect to the same DB. When dev updated the DB, would prod (that runs older code) still works correctly with the upgraded DB? I haven't think through all the features yet, but e.g. registering new user may not work because the `salt` column is `NOT NULL`.

Reply:
> I think we need to use transactions for the public methods of the Update class. There are mulpile replicas of haxe lib server pod running in the cluster.

I've wrapped the methods in transactions. It looks like the `Manager.all()` method uses locking by default, though this only works on mysql, not sqlite, though it seems the server is using mysql anyway?

https://github.com/HaxeFoundation/record-macros/blob/00afa3924e8478203dd500f8b8a6f2174b2f8e13/src/sys/db/Manager.hx#L475-L477
https://github.com/HaxeFoundation/record-macros/blob/00afa3924e8478203dd500f8b8a6f2174b2f8e13/src/sys/db/Manager.hx#L54-L56

> When dev updated the DB, would prod (that runs older code) still works correctly with the upgraded DB?

This would be an issue. Would it be possible to disable the development server while this transition is taking place? Otherwise, I can open a PR to temporarily disable any registrations or other commands that would be a problem. Then we can push it to disable that on the prod server temporarily, perform the migration, and then re-enable everything once it is done.